### ISSUE TITLE
Backtest weird results - Total profit 936559.5%

### ISSUE THREAD

**Issue by dominiclapointe** [freqtrade/freqtrade#5268]
## Describe your environment

  * Operating system: Ubuntu (Windows 10 WSL2)
  * Python Version: Python 3.8.10
  * CCXT version: ccxt==1.51.77
  * Freqtrade Version: freqtrade 2021.6

## Describe the problem:
The backtest results are way out of the possible range

*Explain the problem you have encountered*
As a new user, i am setting up some really basic Strategy to test Freqtrade. My strategy is basically only buy on rsi < x and sell > y. I have run a hyperopt on this strategy and used the proposed value for the backtest. 

### Steps to reproduce:
use these params :
```
INTERFACE_VERSION = 2
    minimal_roi = {
        "0": 0.245,
        "154": 0.166,
        "433": 0.054,
        "1043": 0
    }

    stoploss = -0.348

    trailing_stop = False
    # trailing_only_offset_is_reached = False
    # trailing_stop_positive = 0.01
    # trailing_stop_positive_offset = 0.0  # Disabled / not configured
    timeframe = '1h'

    process_only_new_candles = False

    use_sell_signal = True
    sell_profit_only = False
    ignore_roi_if_buy_signal = False

    startup_candle_count: int = 30

    # Hyperoptable parameters
    buy_rsi = IntParameter(low=1, high=50, default=48, space='buy', optimize=True, load=True)
    sell_rsi = IntParameter(low=50, high=100, default=94, space='sell', optimize=True, load=True)

    # Optional order type mapping.
    order_types = {
        'buy': 'limit',
        'sell': 'limit',
        'stoploss': 'market',
        'stoploss_on_exchange': True
    }

    # Optional order time in force.
    order_time_in_force = {
        'buy': 'fok',
        'sell': 'gtc'
    }
```

with those methods
```
def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
        # RSI
        dataframe['rsi'] = ta.RSI(dataframe)

        return dataframe

    def populate_buy_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
        dataframe.loc[
            (
                # Signal: RSI crosses above 30
                (qtpylib.crossed_above(dataframe['rsi'], self.buy_rsi.value)) & (dataframe['volume'] > 0) 
            ),
            'buy'] = 1

        return dataframe

    def populate_sell_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
        dataframe.loc[
            (
                # Signal: RSI crosses above 70
                (qtpylib.crossed_above(dataframe['rsi'], self.sell_rsi.value)) & (dataframe['volume'] > 0)
            ),
            'sell'] = 1
        return dataframe
```
  
And this config
```
"exchange": {
        "name": "binance",
        "key": "",
        "secret": "",
        "ccxt_config": {"enableRateLimit": true},
        "ccxt_async_config": {
            "enableRateLimit": true,
            "rateLimit": 200
        },
        "pair_whitelist": [
            ".*/USDT"
        ],
        "pair_blacklist": [
            "BNB/BTC",
            "BNB/BUSD",
            "BNB/ETH",
            "BNB/EUR",
            "BNB/NGN",
            "BNB/PAX",
            "BNB/RUB",
            "BNB/TRY",
            "BNB/TUSD",
            "BNB/USDC",
            "BNB/USDS"
        ]
    },
    "pairlists": [
        {"method": "StaticPairList"}
    ],
```

And finally this command

> freqtrade backtesting --config user_data/config.json --strategy MyFirstStrategy --timerange 20210601-20210712 -i 1h

### Observed Results:

  * What happened? Results out of possible range

> ============================================================== BACKTESTING REPORT ==============================================================
|           Pair |   Buys |   Avg Profit % |   Cum Profit % |   Tot Profit USDT |   Tot Profit % |      Avg Duration |   Win  Draw  Loss  Win% |
|----------------+--------+----------------+----------------+-------------------+----------------+-------------------+-------------------------|
|   ETHDOWN/USDT |      4 |     2339047.68 |     9356190.71 |       9365546.900 |      936554.69 |    1 day, 8:15:00 |     3     1     0   100 |

> Backtesting from       | 2021-06-02 06:00:00      |
| Backtesting to         | 2021-07-12 00:00:00      |
| Max open trades        | 10                       |
|                        |                          |
| Total/Daily Avg Trades | 113 / 2.9                |
| Starting balance       | 1000.000 USDT            |
| Final balance          | 9366595.049 USDT         |
| Absolute profit        | 9365595.049 USDT         |
| Total profit %         | 936559.5%                |
| Avg. stake amount      | 100.000 USDT             |
| Total trade volume     | 11300.000 USDT

  * What did you expect to happen? Results that would make sense

### Relevant code exceptions or logs

Note: Please copy/paste text of the messages, no screenshots of logs please.

  ```
  // paste your log here
  ```
Being quite new to the platform, feel free to ask for more information if i didnt provide enough. Thank you

**Comment by dominiclapointe** [comment]
If i do blacklist ETHDOWN/USDT, the results are back to normal (loss 👍). Ive tried deleting the data file and downloading again using `freqtrade download-data -p ETHDOWN/USDT --timerange 20210601-20210712 -t 1h` and i still get the same results.

So if i exclude ETHDOWN, the results make sense but as soon as i include it, results get nuts...

**Comment by xmatthias** [comment]
certain pairs (especially leveraged pairs) do have some excessive / strange behaviour / movements.

Your best bet is to use `plot-dataframe` for this pair and look at the chart - the result is almost certainly correct based on the data - BUT could be caused by a pricing adjustment / scaling (or just look at the chart in question on the exchange).

![2021-07-13-063403_716x543_scrot](https://user-images.githubusercontent.com/5024695/125391198-54190680-e3a4-11eb-84bc-d0a65907a075.png)

Freqtrade won't have this information available (we look at it and it seems obvious) - and as such - will treat it as "normal" behaviour - just as indicators on tradingview charts do.

**Comment by dominiclapointe** [comment]
Ok thanks!
